### ISSUE TITLE
[似合い] Request: order similar Kanji by level/availability

### ISSUE THREAD

**Issue by marciska** [mwil/wanikani-userscripts#22]
Right now, the list of similar Kanji feels rather random.
Kanji that are not yet unlocked are mixed between Kanjis that have been already learned.
This results in big eye jumps especially on large screens, making it a bit harder to **spot differences between Kanji**.

<img width="1566" alt="" src="https://user-images.githubusercontent.com/59295964/212358556-a7a34823-f849-4dce-924d-a92f47a819bd.png">

## Proposal
Order list to show already learned items first, then locked ones after.
I can think of 2 ways:
1. check if Kanji is yet unlocked, and if not, put it at the end
2. order by WaniKani-level

**Comment by mwil** [comment]
Currently, the kanji are sorted by a "similarity score", so the ones one the left should be the easiest to confuse. I'm not against moving the not yet unlocked ones to the end, but ordering by WK level would break the "pin kanji of your choice to the top" functionality, because atm it just adds a big boost to the score.

**Comment by marciska** [comment]
You are right. I made a pull request a while ago, but changed it now according to your suggestion. Sorting into two groups of (A: unlocked Kanji, B: locked Kanji), and sorting both groups by their score rather than their level makes more sense to me.
In any way, the gaps between kanji I learned and have not yet learned are quite distracting for me, but I'm happy to accept everyone else's opinion.
